1.10/deploying-services/gpu.md

```diff
@@ -0,0 +1,175 @@
+---
+post_title: Using GPUs
+feature_maturity: preview
+menu_order: 110
+---
+
+DC/OS supports allocating GPUs (Graphics Processing Units) to your long-running DC/OS services. Adding GPUs to your services can dramatically accelerate big data workloads. 
+
+With GPU-based scheduling, you can share cluster resources for traditional and machine learning workloads, as well as dynamically allocate GPU resources inside those clusters and free them when needed. You can reserve GPU resources for the workloads that need them, or pool these GPU-enabled resources with the rest of the infrastructure for higher overall utilization.
+
+After installing DC/OS with GPUs enabled, you can specify GPUs in your application definitions with the `gpus` parameter.
+
+# Installing DC/OS with GPUs Enabled
+GPUs must be enabled during DC/OS installation. Follow the instructions to enable GPUs based on your DC/OS deployment method.
+
+## Custom DC/OS Installation with GPUs
+
+1.  Install the [NVIDIA Management Library (NVML)](https://developer.nvidia.com/nvidia-management-library-nvml) on each node of your cluster that has GPUs, unless it is already installed. Find detailed installation instructions [here](https://github.com/apache/mesos/blob/master/docs/gpu-support.md#external-dependencies).
```

Comment:
The document linked mentions it briefly, but it's probably also worth mentioning here that the minimum required Nvidia driver version is 340.29.